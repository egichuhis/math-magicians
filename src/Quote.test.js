import React from 'react';
import { render } from '@testing-library/react';
import ShowQuote from './components/quote/showQuote';

test('ShowQuote renders correctly', () => {
  const { container } = render(<ShowQuote />);
  expect(container).toMatchSnapshot();
});
