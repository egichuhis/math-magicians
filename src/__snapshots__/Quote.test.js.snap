// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowQuote renders correctly 1`] = `
<div>
  <div
    class="quoteDiv"
  >
    <p>
      Loading...
    </p>
  </div>
</div>
`;
